packages/fxa-settings/src/components/BentoMenu/index.tsx

```diff
@@ -35,7 +35,7 @@ export const BentoMenu = () => {
         title={l10n.getString('bento-menu-title')}
         aria-expanded={isRevealed}
         aria-controls={dropDownId}
-        className="rounded p-1 w-7 mx-2 border-transparent hover:bg-grey-200 focus:outline-none transition-standard desktop:mx-8"
```

Comment:
@meandavejustice it looks like you authored these classes - any reason why we shouldn't use a focus outline? Testing on my end looks good but I might be missing some context. What you think?

https://user-images.githubusercontent.com/6392049/113728269-efbf1a80-96c3-11eb-9ab2-814c14ce40b9.mov

Reply:
Focus outline looks good 👍 👍, Probably just copy-pasta'd the `outline:none`